apps/api/src/app/content-templates/usecases/compile-template/compile-template.usecase.ts

```diff
@@ -44,6 +44,8 @@ export class CompileTemplate {
 
     const template = Handlebars.compile(templateContent);
 
-    return template(command.data);
+    const result = template(command.data, {});
+
+    return result.replace(/&#x27;/g, "'");
```

Comment:
@davidsoderberg are we sure its only for ' ?
What about lets say &, I think that one also has a problem. And maybe others?